types/office-js-preview/index.d.ts

```diff
@@ -23048,7 +23050,7 @@ declare namespace Office {
          * @param callback - When the method completes, the function passed in the `callback` parameter is called with a single parameter,
          *                `asyncResult`, which is an `Office.AsyncResult` object.
          */
-        getAllAsync(callback: (asyncResult: Office.AsyncResult<string>) => void): void;
+        getAllAsync(callback: (asyncResult: Office.AsyncResult<object>) => void): void;
```

Comment:
If this is being changed to an `object`, should the set methods still take in a string and use terms like "characters"?

Reply:
Yes. The set methods still take in a string to define a key and value.